src/DroneSafetyApi/Services/IHazardsToHeatmapsResponse.cs

```diff
@@ -4,8 +4,21 @@
 
 namespace DroneSafetyApi.Services
 {
+    /// <summary>
+    /// The IHazardsToHeatmapsResponse interface is capable of generating <see cref="HeatmapsResponse"/>s
```

Comment:
Say that this is the most general interface, which must be implemented by any strategy created a HeatmapsResponse from a Hazard list